rq/queue.py

```diff
@@ -23,6 +24,30 @@ def compact(lst):
     return [item for item in lst if item is not None]
 
 
+class JobEnqueueData(namedtuple('JobEnqueueData', ["func", "args", "kwargs", "timeout",
```

Comment:
Is there a reason why we're naming this `JobEnqueueData` as opposed to `JobData`? Also open for other ideas.

Reply:
Thinking further about this, you're right that `JobEnqueueData` makes more sense since this is not just simply data about job itself. Just wondering if there's a better name for this. Some ideas:
```python
queue.enqueue_many(
    queue.EnqueueData(func=foo, result_ttl=60),
    queue.EnqueueData(func=bar)
)
```

queue.enqueue_many(
    queue.prepare_data(func=foo, result_ttl=60),
    queue.prepare_data(func=bar)
)
```

Reply:
I'm fine with either.  You could argue the latter one is better because it does not expose the inner type.

Reply:
Ok, then let's go with 
```python
queue.enqueue_many(
    queue.prepare_data(func=foo, result_ttl=60),
    queue.prepare_data(func=bar)
)
```

Also open to suggestion on a better name for `prepare_data`.

Reply:
@selwin I don't have any strong feelings on the name, `prepare_data` honestly does not seem bad to me.  I'll implement it that way, at least for now.